README.md

```diff
@@ -1,296 +1,93 @@
-- [Delete data set member and a data set](#delete-a-pds-member-and-pds): Delete a chosen data set member or an entire data set.
-- [View and access multiple profiles simultaneously](#view-and-access-multiple-profiles-simultaneously): Work with data sets from multiple profiles.
-- [Allocate Like](#allocate-like): Create a copy of a chosen data set with the same parameters.
-
-### View data sets and use multiple filters
-
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Hover over the profile that you want to apply the filter to.
-4. Click the **magnifying glass** icon.
-5. Enter a pattern you want to create a filter for.
-   The data sets that match your pattern(s) are displayed in the explorer tree.
-
-**Tip:** To provide multiple filters, separate entries with a comma. You can append or postpend any filter with an \*, which indicates wildcard searching. You cannot enter an \* as the entire pattern.
-
-![View Data Set](/docs/images/ZE-multiple-search.gif?raw=true "View Data Set")
-<br /><br />
-
-#### View data sets with member filters
-
-![View Data Set With Member Pattern](/docs/images/ZE-member-filter-search.gif?raw=true "View Data Set With Member Pattern")
-
-**Note:** You cannot favorite a data set or member that includes a member filter search pattern.
-<br /><br />
-
-### Refresh the list of data sets
-
-1. Navigate to the explorer tree.
-2. Click **Refresh All** button (circular arrow icon) on the right of the **DATA SETS** explorer bar.
-
-### Rename data sets
-
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Select a data set you want to rename.
-4. Right-click the data set and select the **Rename Data Set** option.
-5. Change the name of the data set.
-
-![Rename Data Set](/docs/images/ZE-rename.gif?raw=true "Rename Data Set")
-<br /><br />
-
-### Copy data sets
-
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Select a member you want to copy.
-4. Right-click the member and select the **Copy Data Set** option.
-5. Right-click the data set where the member belongs and select the **Paste Data Set** option.
-6. Enter the name of the copied member.
-
-![Copy Data Set](/docs/images/ZE-copy.gif?raw=true "Copy Data Set")
-<br /><br />
-
-### Download, edit, and upload existing PDS members
-
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Open a profile.
-4. Click the PDS member (or PS) that you want to download.
-
-   **Note:** To view the members of a PDS, click the PDS to expand the tree.
-
-   The PDS member is displayed in the text editor window of VSC.
-
-5. Edit the document.
-6. Navigate back to the PDS member (or PS) in the explorer tree, and click the **Save** button.
-
-Your PDS member (or PS) is uploaded.
-
-**Note:** If someone else has made changes to the PDS member (or PS) while you were editing it, you can merge your conflicts before uploading to the mainframe.
+- Install [Yarn](https://yarnpkg.com/getting-started/install).
```

Comment:
This section might be clearer if we group the client-side prerequisites together separately from the host prerequisites instead of mixing the two together. Node.js and Yarn will be needed locally on the client-side. The (current) middle bullet point about TSO/E, etc. is host-side.